addon/components/light-table.js

```diff
@@ -219,6 +201,11 @@ const LightTable = Component.extend({
     if (isNone(media)) {
       this.set('responsive', false);
     }
+
+    // wait until after view's element has rendered in order to query the column elements
+    Ember.run.scheduleOnce('afterRender', () => {
```

Comment:
I'm not sure of the best way to wait until the view's `element` is ready to be queried here. Open to ideas.